### PR TITLE
add Zalando to Case Study index page

### PR DESCRIPTION
**Pull request by alexcontini**

### CODE REVIEW

File: case-studies/index.html [hunk 1]
```diff
@@ -32,12 +40,7 @@ <h5>A collection of users running Kubernetes in production.</h5>
                   <!--<p class="attrib">—  Martin Danielsson, Solution Architect at Haufe Group</p>-->
                   <a href="/case-studies/haufegroup/">Read about Haufe Group</a>
               </div>
-              <div class="case-study">
-                  <img src="/images/case_studies/blackrock.png" alt="BlackRock">
-                  <p class="quote">"My message to other enterprises like us is you can actually integrate Kubernetes into an existing, well-orchestrated machinery."</p>
-                  <!--<p class="attrib">— Michael Francis, Managing Director at BlackRock</p>-->
-                  <a href="/case-studies/blackrock/">Read about BlackRock</a>
-              </div>
```
Comment (tengqm) [inline case-studies/index.html]: why remove blackrock?

Comment (alexcontini) [reply case-studies/index.html]: @tengqm sorry I thought for some reason to only include 4 there so I have been removing the oldest one of the 4. I will add blackrock back in

Comment (Bradamant3) [reply case-studies/index.html]: The page layout looks as though it accommodates only 4, and I've always seen only four on that page. I don't think this is something that sig-docs needs to weigh in on -- it's my understanding that the marketing and communications folks manage the content for this and related pages.

**Comment by Bradamant3** [comment]
Going ahead and getting merged without adding blackrock. If we need to add it back in, apologies @alexcontini !

/approve
/lgtm

**Pull request merged**